src/Form/CitizenProjectCommandType.php

```diff
@@ -55,10 +65,38 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->addEventListener(
                 FormEvents::PRE_SET_DATA, [$this, 'preSetData']
-            )
-        ;
+            );
 
         $builder->get('address')->remove('address');
+
+        /** @var CitizenProjectCommand $citizenProjectCommand */
+        $citizenProjectCommand = $builder->getData();
+
+        if (
+            $citizenProjectCommand instanceof CitizenProjectCreationCommand
+            || $citizenProjectCommand->getCitizenProject()->isPending()
```

Comment:
The `||` is still broken here, shouldn't it be `&&`?

Reply:
No, I will see committees and committees_search on creation or during the pending status of CitizenProject 'before approve by admin)

Reply:
```php
if (
    $citizenProjectCommand instanceof CitizenProjectCreationCommand
    || $citizenProjectCommand->getCitizenProject()->isPending()
```
If the first statement is `true` the second one is not evaluated.
If the first statement is false `getCitizenProject` will break